internal/operands/common-templates/reconcile.go

```diff
@@ -42,6 +42,11 @@ func GetOperand() operands.Operand {
 	return &commonTemplates{}
 }
 
+const (
+	operandName      = "common-templates"
```

Comment:
Those are currently conflicting with https://github.com/kubevirt/ssp-operator/pull/73/files#diff-f6d56152bde2c23e8384a420483683cbfc8b9d3152bd9c104914c737b8a99cf3R45.
I think I'd prefer it as part of the interface as well but didn't want to extend it just for that.
But the naming scheme is a bit different so we should agree on something, I think this format adheres more to the general style in kubernetes naming and labels.